src/OXRS_MQTT.cpp

```diff
@@ -114,6 +118,8 @@ void OXRS_MQTT::loop()
 
 void OXRS_MQTT::receive(char * topic, uint8_t * payload, unsigned int length) 
 {
+  _screen->trigger_mqtt_rx_led ();

```

Comment:
Think we need to allow `_screen` to be NULL in case someone doesn't have one. So add NULL checks around these calls I think?

Reply:
done